src/test/e2e/tests/specs.test.ts

```diff
@@ -113,11 +107,7 @@ describe('Unverified user', () => {
 
       it('selecting first result should take you to to the hearings list page', async () => {
         hearingListPage = await alphabeticalSearchPage.selectFirstListResult();
-        expect(await hearingListPage.getPageTitle()).toEqual('Blackburn Magistrates\' Court hearing list');
-      });
-
-      it(`should display ${expectedNumOfHearings} results`, async() => {
-        expect(await hearingListPage.getResults()).toBe(expectedNumOfHearings);
+        expect(await hearingListPage.getPageTitle()).toEqual('Summary of Publications for Blackburn Magistrates\' Court');
```

Comment:
youve still got the old flow in here, this isnt the hearing list page, theres a full new page added here that sits in the flow which means the test of the number of results should be moved further along after something is clicked?